src/test/regress/expected/create_role_propagation.out

```diff
@@ -240,10 +242,11 @@ SELECT roleid::regrole::text AS role, member::regrole::text, (grantor::regrole::
       role       |     member      | grantor | admin_option
 ---------------------------------------------------------------------
  dist_role_1     | dist_role_2     | t       | f
- dist_role_3     | non_dist_role_3 | t       | f
+ dist_role_3     | non_dist_role_3 | f       | f
```

Comment:
Can we also add another test to make sure that we properly set grantor on other nodes too, maybe by using:

```sql
SELECT result FROM run_command_on_all_nodes(
  $$
  SELECT json_agg(q.* ORDER BY member) FROM (
    SELECT member::regrole::text, grantor::regrole::text, admin_option
    FROM pg_auth_members WHERE roleid::regrole::text = 'dist_role_3'
  ) q;
  $$
);
```

Reply:
Added run_all_nodes into the query

Reply:
Added the test to see the grantor for the new statement.